web/satellite/vuetify-poc/src/components/dialogs/createAccessSteps/ChoosePermissionsStep.vue

```diff
@@ -67,34 +98,53 @@
                     <template #prepend-item>
                         <v-list-item
                             title="All buckets"
-                            color="primary"
                             :active="isAllBucketsSelected"
+                            density="compact"
                             @click="isAllBucketsSelected = !isAllBucketsSelected"
                         >
                             <template #prepend>
-                                <v-checkbox-btn v-model="isAllBucketsSelected" color="primary" />
+                                <v-checkbox-btn v-model="isAllBucketsSelected" />
                             </template>
                         </v-list-item>
                         <v-divider />
                     </template>
 
                     <template #item="{ props: slotProps }">
-                        <v-list-item v-bind="slotProps" color="primary">
+                        <v-list-item v-bind="slotProps" density="compact">
                             <template #prepend="{ isSelected }">
-                                <v-checkbox-btn :model-value="isSelected" color="primary" />
+                                <v-checkbox-btn :model-value="isSelected" />
                             </template>
                         </v-list-item>
                     </template>
                 </v-autocomplete>
+
+                <!-- <v-chip-group
+                    v-model="buckets"
+                    column
+                    multiple
+                    class="mb-3"
+                >
+                    <v-chip
+                        class="text-body-2"
+                        rounded="xl"
+                        filter
+                        variant="outlined"
+                        value="All Buckets"
+                    >
+                        All Buckets
+                    </v-chip>
+                </v-chip-group> -->
```

Comment:
NIT - this can also be removed

Reply:
This is left for reference as part of next update for the access grants flow.